pkg/utils/k8sutil.go

```diff
@@ -750,15 +757,23 @@ func splitHandler(handler string) (string, string, error) {
 }
 
 func ensureFuncConfigMap(client kubernetes.Interface, funcObj *spec.Function, or []metav1.OwnerReference) error {
-	modName, _, err := splitHandler(funcObj.Spec.Handler)
-	if err != nil {
-		return err
-	}
+	configMapData := map[string]string{}
+	var err error
+	if funcObj.Spec.Handler != "" {
```

Comment:
This one as well. In case of using custom runtime, the runtime field might be empty that will cause the error

Reply:
I added the validation with the code command until I get the confirmation for enabling it and removing the related part in the documentation.